iree/compiler/Codegen/LLVMGPU/test/vectorization.mlir

```diff
@@ -50,7 +50,7 @@ func @add_dispatch_0() attributes {cuda_workgroup_size = dense<[32, 1, 1]> : vec
   return
 }
 // CHECK-LABEL: func @add_dispatch_0()
-//       CHECK:   vector.transfer_read {{.*}} : memref<1024x1024x1024xf32>, vector<4xf32>
-//       CHECK:   vector.transfer_read {{.*}} : memref<1024x1024x1024xf32>, vector<4xf32>
+//       CHECK:   vector.transfer_read {{.*}} : memref<1x1x4xf32, {{.*}}>, vector<1x1x4xf32>
```

Comment:
Can you explain a bit about why the type of the `memref` changed here. Its no immediately obvious to me. Also for clarity might be good to just check the surrounding scope instead of the instruction directly.

Reply:
I moved the `populateFoldSubViewOpPatterns` later in the pipeline as it makes optimizing on vector transfer easier so now the memref is the same as the input test IR.